wordpress-mobile/WordPress-Android
# Remove follow conversation from notification FF and related obsolete code.

This PR removes `FOLLOW_BY_PUSH_NOTIFICATION` feature flag and Follow Conversation button from the reader post comment list header since it moved into the AppBar.

To test:
- Navigate to reader post comments.
- Confirm that there is no Follow Conversation button in the header view.
- Try following the conversation from AppBar and confirm that it works.

## Regression Notes
1. Potential unintended areas of impact
Follow conversation being broken somewhere we do not expect it.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Existing tests + manual testing.

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
